crates/ruff_linter/src/rule_redirects.rs

```diff
@@ -103,6 +103,16 @@ static REDIRECTS: Lazy<HashMap<&'static str, &'static str>> = Lazy::new(|| {
         ("TRY200", "B904"),
         ("PGH001", "S307"),
         ("PGH002", "G010"),
+        // flake8-trio and flake8-async merged with name flake8-async
+        ("TRIO", "ASYNC"),
+        ("TRIO1", "ASYNC1"),
+        ("TRIO10", "ASYNC10"),
+        ("TRIO100", "ASYNC100"),
+        ("TRIO105", "ASYNC105"),
+        ("TRIO109", "ASYNC109"),
+        ("TRIO11", "ASYNC11"),
+        ("TRIO110", "ASYNC110"),
+        ("TRIO115", "ASYNC115"),
```

Comment:
It looks like some ASYNC rules were also moved from one code to another.

Reply:
ASYNC100 moved to ASYNC210, (but the ASYNC100 code was taken over by TRIO100)
ASYNC101 was split into ASYNC220, ASYNC221, ASYNC222, ASYNC230, and ASYNC251
ASYNC102 was merged into ASYNC220 and ASYNC221

Reply:
@charliermarsh any advice on this?

Reply:
I think what you have here is right... The TRIO rules are gone, so lets redirect them away. But the ASYNC rules probably can't be safely redirected.